genesyscloud/flow_loglevel/resource_genesyscloud_flow_loglevel_utils.go

```diff
@@ -0,0 +1,83 @@
+package flow_loglevel
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/mypurecloud/platform-client-sdk-go/v125/platformclientv2"
+	"terraform-provider-genesyscloud/genesyscloud/util/resourcedata"
+)
+
+func generateFlowLogLevelResource(
```

Comment:
This function can be in the test file. We only need to have these functions outside of the test file when they are public since functions can't be public in a test file. In this case though since the function is private it can stay in the test file